src/protocol/requests/joinGroup/v3/request.js

```diff
@@ -0,0 +1,33 @@
+const requestV1 = require('../v1/request')
+
+/**
+ * JoinGroup Request (Version: 3) => group_id session_timeout rebalance_timeout member_id protocol_type [group_protocols]
+ *   group_id => STRING
+ *   session_timeout => INT32
+ *   rebalance_timeout => INT32
+ *   member_id => STRING
+ *   protocol_type => STRING
+ *   group_protocols => protocol_name protocol_metadata
+ *     protocol_name => STRING
+ *     protocol_metadata => BYTES
+ */
+
+module.exports = ({
+  groupId,
+  sessionTimeout,
+  rebalanceTimeout,
+  memberId,
+  protocolType,
+  groupProtocols,
+}) =>
+  Object.assign(
+    requestV1({
```

Comment:
The V2 request is apparently identical to V1, so it doesn't really matter from a practical perspective, but just for symmetry it would be nice to have v3 import from v2.